react.js

```diff
@@ -0,0 +1,71 @@
+module.exports = {
+	extends: ["airbnb", "airbnb/hooks"],
+	plugins: [],
+	env: {
+		es6: true,
+		node: true,
+		browser: true,
+	},
+	parserOptions: {
+		ecmaVersion: 6,
```

Comment:
Will we be able to use newer features like optional chaining operator with that setting? I think it's from newer ES versions, not 6.

Reply:
the latest is the 12 if im not wrong, we can use it. I will change this line